Microsoft.Toolkit.HighPerformance/Extensions/BoolExtensions.cs

```diff
@@ -58,7 +65,8 @@ public static unsafe int ToInt(this bool flag)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static unsafe int ToBitwiseMask32(this bool flag)
         {
-            byte rangeFlag = *(byte*)&flag;
+            bool copy = flag;
```

Comment:
Curious as to the change here? Just for my own understanding. 🙂

Reply:
NVM missed the comment above. Should this be a compile optimization issue to file against .NET? @Sergio0694

Reply:
It's a known issue - they have explicit workarounds for this inside the runtime as well. For now we just need to remember to do this and work around it, thankfully the solution is easy enough, it's just not as clean as doing this directly. Will ask if there's any news on this though, we were just about talking about this a few days ago in the C# server 😄